plugin/gradle/libs.versions.toml

```diff
@@ -5,21 +5,21 @@ androidPlugin = "4.1.0"
 semver = "1.1.1"
 jgit = "5.6.0.201912101111-r"
 sl4fj = "1.7.30"
-gradleWrapper = "7.1.1"
-gradleWrapperSha = "33ad4583fd7ee156f533778736fa1b4940bd83b433934d1cc4e9f608e99a6a89"
+gradleWrapper = "7.2"
+gradleWrapperSha = "f581709a9c35e9cb92e16f585d2c4bc99b2b1a5f85d2badbd3dc6bff59e1e6dd"
```

Comment:
wrapper jar is wrong, should be `e996d452d2645e70c01c11143ca2d3742734a28da2bf61f25c82bdc288c9e637`

Reply:
It should belong to 6.9.1 ?

![image](https://user-images.githubusercontent.com/10363352/133433045-644edef9-e337-43c3-88e8-09612db71a3f.png)

Reply:
Is the `gradleWrapperSha` should be gradle-distribution-sha instead of gradle-wrapper-jar-sha ? We shoud rename it.

Reply:
No need to rename and it should stay `33ad4583fd7ee156f533778736fa1b4940bd83b433934d1cc4e9f608e99a6a89`. Seems it was not changed between 7.1.1 and 7.2 releases.